engine/runtime-with-polyglot/src/test/java/org/enso/interpreter/test/JsInteropTest.java

```diff
@@ -0,0 +1,45 @@
+package org.enso.interpreter.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Value;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class JsInteropTest extends TestBase {
+
+  private static final ByteArrayOutputStream out = new ByteArrayOutputStream();
+  private Context ctx;
+
+  @Before
+  public void initContext() {
+    ctx = createDefaultContext(out);
+    out.reset();
+  }
+  
+  @After
+  public void disposeCtx() {
+    ctx.close();
+  }
+
+  @Test
+  public void testDefaultJSPrint() {
+    var src = """
```

Comment:
This test cannot be written in pure Enso - we need to invoke the `main` method externally. The `main` method returns `PolyglotProxy` for a foreign JS object, and we check that it is correctly converted to `Text` here.